src/ApplicationLogs/LogReader.cs

```diff
@@ -30,7 +30,14 @@ protected override void Configure()
         protected override void OnSystemLoaded(NeoSystem system)
         {
             if (system.Settings.Network != Settings.Default.Network) return;
-            RpcServerPlugin.RegisterMethods(this, Settings.Default.Network);
+            try
+            {
+                RpcServerPlugin.RegisterMethods(this, Settings.Default.Network);
+            }
+            catch
+            {
+                Console.WriteLine("Please run `install RpcServer` to install RpcServer Plugin.");
```

Comment:
Just please install? Maybe the command change in a future

Reply:
Just in case, it is not expected to be triggered.

Reply:
I think we don't need this `try`.

Reply:
> I think we don't need this `try`.

You can never make sure that the dependency always exists, if they don't, `file not exists exception` will throw and the software will crash directly without telling users how to fix it.